db/crud.go

```diff
@@ -1571,7 +1571,7 @@ func (db *DatabaseCollectionWithUser) assignSequence(ctx context.Context, docSeq
 		// on the feed is small - sub-second, so we usually shouldn't care about more than
 		// a few recent sequences.  However, the pruning has some overhead (read lock on nextSequence),
 		// so we're allowing more 'recent sequences' on the doc (20) before attempting pruning
-		stableSequence := db.changeCache.GetStableSequence(doc.ID).Seq
+		stableSequence := db.changeCache().GetStableSequence(doc.ID).Seq
```

Comment:
It looks like this entire function (assignSequences) could get switched back to having a *databaseContext receiver.  Again, that could be part of a subsequent clean-up, but will help future reasoning about these functions to have them appropriately scoped.

Reply:
fixed, I agree about clarity.